mscore/loginmanager.cpp

```diff
@@ -418,6 +418,8 @@ void LoginManager::onGetMediaUrlRequestReady(QByteArray ba)
                         }
                   }
             preferences.exportMp3BitRate = br;
+            // delete the temp file created above
+            QFile::remove(path);
```

Comment:
Mhh, I don't think the OS will clean this up and outside path isn't know anymore, or  is it?